Tasks/UsePythonVersionV0/usepythonversion.ts

```diff
@@ -52,8 +52,8 @@ function pypyNotFoundError(majorVersion: 2 | 3) {
 // For example, PyPy 7.0 contains Python 2.7, 3.5, and 3.6-alpha.
 // We only care about the Python version, so we don't use the PyPy version for the tool cache.
 
-function usePyPy(majorVersion: 2 | 3, parameters: TaskParameters, platform: Platform): void {
-    const findPyPy = tool.findLocalTool.bind(undefined, 'PyPy', majorVersion.toString());
+function usePyPy(majorVersion: '2' | '3.6', parameters: TaskParameters, platform: Platform): void {
```

Comment:
Nit: since it's no longer major version - could you please rename it to versionSpec?

Reply:
Done